src/features/landing/components/ForWhoSection/ForWhoSectionCards/ForWhoSectionCards.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Grid, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import useScreenSize from '../../../hooks/useScreenSize/useScreenSize';
+import { TRANSLATION_NAMESPACE } from '../../../utils/constants/constants';
+
+const styles = {
+  mainBox: {
+    position: 'relative',
+    top: '-13.4375rem', // -215px
+    padding: '0 0 0 0',
+  },
+  secondaryHeading: {
+    maxWidth: '374px',
+    width: '100%',
+    color: '#1A1C1E',
+    fontFamily: 'Stolzl-Regular, sans-serif',
+    fontSize: '28px',
+    fontStyle: 'normal',
+    fontWeight: 700,
+    lineHeight: 'normal',
+  },
+};
+
+export default function ForWhoSectionCards({ cardItemsJSX }: { cardItemsJSX: React.ReactNode }) {
+  const { t } = useTranslation(TRANSLATION_NAMESPACE);
+  const { isSmallest } = useScreenSize();
+
+  return (
+    <Box sx={{ ...styles.mainBox }}>
+      <Grid item>
+        <Typography
+          style={{
+            ...styles.secondaryHeading,
+            fontSize: isSmallest ? '22px' : styles.secondaryHeading.fontSize,
+          }}
+        >
+          {t('for_who.heading_secondary')}
+        </Typography>
+      </Grid>
+      <Grid
+        container
+        alignItems="stretch"
+        spacing={3}
+        sx={{
+          position: 'absolute',
+          bottom: '-150px',
```

Comment:
for what reason you do like you do? in the future it can broke our layout, try find another solution without negative values